rule-tester-setup.ts

```diff
@@ -0,0 +1,9 @@
+import { RuleTester } from "@typescript-eslint/rule-tester";
+import * as vitest from "vitest";
```

Comment:
We need a wildcard import?

Reply:
You are right! not needed and replaced

Reply:
Coolio. Actually, is there an eslint rule to prevent `import * as`?